src/docfx/serve/lsp/LanguageServerHost.cs

```diff
@@ -21,38 +21,40 @@ internal class LanguageServerHost
             ILanguageServerNotificationListener? notificationListener = null)
         {
             using var cts = new CancellationTokenSource();
+            using (Log.BeginScope(commandLineOptions.Verbose))
```

Comment:
Move it to `StartLanguageServer` method because it is AspNetCore that lost this context. (Which means other APIs there in the future also needs to call `Log.BeginScope`)